lib/backend.js

```diff
@@ -833,6 +844,16 @@ Backend.prototype._fetchSnapshotByTimestamp = function(collection, id, timestamp
   var from = 0;
   var to = null;
 
+  var shouldGetLatestSnapshot = timestamp === null;
+  if (shouldGetLatestSnapshot) {
+    var options = {metadata: true};
```

Comment:
Same here, why are we fetching metadata?

Reply:
Same as above